Dockerfile

```diff
@@ -25,16 +25,10 @@ RUN pip uninstall easypqp \
     && pip install git+https://github.com/Nesvilab/easypqp.git@master \
     && pip install lxml
 	
-# set the working directory to the HOME of root
-WORKDIR /root
```

Comment:
In the container, the user is `root`. I think there is no harm to change to the HOME directory.